FAQ for difference between sample and library ID (#32)
AlexsLemonade/scpca-docs: docs/faq.md

scpca_sample <- readRDS("SCPCL000000_filtered.rds")
```

## What is the difference between participants, samples, and libraries?

> The `participant_id` indicates the participant, or patient, from which a sample, or collection of samples, is obtained from. 

Reviewer: We don't have participant ids for all samples right? Only when there is more than one sample from the same participant? If so, we should say that explicitly.  (Maybe in a separate FAQ entry)

I simplified here, mostly because there were lots of commas that I think were getting confusing.
```suggestion
The `participant_id` indicates the participant from which a sample or collection of samples was obtained. 
```

A sample ID, labeled as `scpca_sample_id` and indicated by the prefix `SCPCS`, represents a unique tissue that was collected from a participant. 
For example, one participant may have a sample collected both at initial diagnosis and at relapse.

> This would result in two unique sample ID's with the same participant ID. 

Reviewer: Only because you just said "unique"
```suggestion
This would result in two different sample ID's with the same participant ID. 
```

> The library ID, labeled as `scpca_library_id` and indicated by the prefix `SCPCL`, represents a single emulsion and droplet generation using the 10X Genomics workflow from a given sample. 

Reviewer: I think we want to make this a bit less specific, as we use library ids also for bulk, so we should to say something about that. I'm fighting with it a bit here, but trying to get across that the CITE-seq sequencing library falls in the same library, but bulk does not. I don't know it "set of cells" is a good metric, but it is what I came up with...

```suggestion
The library ID, labeled as `scpca_library_id` and indicated by the prefix `SCPCL`, represents a single set of cells from a tissue sample.
For a bulk RNA-seq experiment, this will result in a single sequencing library. 
For single-cell or single-nuclei experiments, this will be the result of emulsion and droplet generation using the 10X Genomics workflow, potentially including both RNA-seq, CITE-seq and cell hashing sequencing libraries. 
```

> In most cases, each sample will only have one corresponding library.

Reviewer: Realizing that as we get bulk, this may no longer be true.
```suggestion
In most cases, each sample will only have one corresponding single-cell library, and may also have an associated bulk RNA-seq library.
```

> However, in some cases multiple libraries underwent separate droplet generation and sequencing from the same sample, resulting in more than one library ID being associated with the same sample ID. 

Reviewer: Adding a potential "why" here.
```suggestion
However, in some cases multiple libraries were generated from the same sample (perhaps with different experimental protocols), resulting in more than one library ID being associated with the same sample ID. 
```

scpca_sample <- readRDS("SCPCL000000_filtered.rds")
```

## What is the difference between samples and libraries?

A sample ID, labeled as `scpca_sample_id` and indicated by the prefix `SCPCS`, represents a unique tissue that was collected from a participant. 

The library ID, labeled as `scpca_library_id` and indicated by the prefix `SCPCL`, represents a single set of cells from a tissue sample.
For single-cell or single-nuclei experiments, this will be the result of emulsion and droplet generation using the 10X Genomics workflow, potentially including both RNA-seq, CITE-seq and cell hashing sequencing libraries. 
For a bulk RNA-seq experiment, this will result in a single sequencing library. 

In most cases, each sample will only have one corresponding single-cell library, and may also have an associated bulk RNA-seq library.

> However, in some cases multiple libraries were created by separate droplet generation and sequencing from the same sample, resulting in more than one library ID being associated with the same sample ID. 

Reviewer: ```suggestion
However, in some cases multiple libraries were created by separate droplet generation and sequencing from the same sample, resulting in more than one single-cell library ID being associated with the same sample ID. 
```

## Why do some samples have missing participant IDs?

> The `participant_id` indicates the participant from which a collection of samples was obtained. 

Reviewer: Emphasizing again at the start that this is a conditional field.
```suggestion
The `participant_id`, when present, indicates the participant from which a collection of samples was obtained. 
```

For example, one participant may have a sample collected both at initial diagnosis and at relapse.
This would result in two different sample ID's, but the same participant ID. 

> For the most part, each participant only has one corresponding sample that was collected and submitted for sequencing. 

Reviewer: ```suggestion
However, for most participants, only a single sample was collected and submitted for sequencing. 
```

> Because of this, many of the samples do not have a participant ID and only those samples in which multiple unique samples have been obtained from the same participant will a participant ID be present. 

Reviewer: Separating the second clause and making it more active.
```suggestion
Because of this, many of the samples do not have a separate participant ID. 
Participant IDs are only present for samples that were derived from the same participant as at least one other sample. 
```
